app/services/validation_beta_service.rb

```diff
@@ -1,6 +1,19 @@
 # frozen_string_literal: true
 
 class ValidationBetaService
+  def remind_sits_to_add_participants
+    empty_school_cohorts = SchoolCohort.where.not(id: ParticipantProfile::ECF.select(:school_cohort_id))
+    School.where(id: empty_school_cohorts.select(:school_id)).includes(:induction_coordinators).find_each do |school|
+      school.induction_coordinators.each do |sit|
+        SchoolMailer.remind_sits_to_setup_cohort(
+          recipient: sit.email,
+          school_name: school.name,
+          campaign: :sit_to_complete_steps,
+        ).deliver_later
+      end
+    end
+  end
```

Comment:
Soo, question one, if an IC has multiple schools, should they get more than one email? In other cases we tend to just send them one, but this will not be so kind.

Question two, schools can opt out - `not_opted_out` scope on schools. Should we respect that?

Question three, school cohort has states like `no_early_career_teachers`, `school_funded_fip`. I am pretty sure we don't collect teachers for those two. Should we exclude those school cohorts?

Reply:
All good questions, will check with Nicola. :)